beanstalk_dispatch/execution.py

```diff
@@ -29,9 +32,21 @@ def execute_function(function_request):
     if function_path:
         # TODO(marcua): Catch import errors and rethrow them as
         # BeanstalkDispatchErrors.
-        function = locate(function_path)
-        function(*function_request[ARGS], **function_request[KWARGS])
+        runnable = locate(function_path)
+        args = function_request[ARGS]
+        kwargs = function_request[KWARGS]
+        if inspect.isclass(runnable):
+            if issubclass(runnable, SafeTask):
+                task = runnable(args=args, kwargs=kwargs)
```

Comment:
before this was mixing the args/kwargs of the task with the args/kwargs of the class so i separated them. not super necessary now but if we refactor later we may have to cahnge it back